regression/pages/studio/course_info_studio.py

```diff
@@ -0,0 +1,61 @@
+"""
+Course Updates page.
+"""
+from edxapp_acceptance.pages.studio.utils import type_in_codemirror
+from edxapp_acceptance.pages.common.utils import click_css
+
+from regression.pages.studio.course_page_studio import CoursePageExtended
+
+
+class CourseUpdatesPageExtended(CoursePageExtended):
+    """
+    Course Updates page.
+    """
+
+    url_path = "course_info"
+
+    def is_browser_on_page(self):
+        return all([self.q(css='body.view-updates').present,
+                    self.q(css='#course-handouts-view .edit-button').visible])
+
+    def open_new_update_form(self):
+        """
+        Open update form
+        """
+        click_css(self, '.button.new-button.new-update-button', 0, False)
+        self.wait_for_element_presence(
+            '.new-update-form', 'Update form has been opened')
+
+    def write_update_and_save(self, new_update):
+        """
+        Write new update and save it.
+        """
+        type_in_codemirror(self, 0, new_update)
+        click_css(self, '.save-button', 0, True)
```

Comment:
Do we need a `wait_for` condition after this click, to ensure the page is done painting and is ready for the assertion or whatever else comes next?